src/redisWrapper.js

```diff
@@ -165,6 +168,14 @@ const getMainClient = async () => {
   return mainClient;
 };
 
+/**
+ * Closes the main Redis client if it is open. Returns any error that occurs during close or undefined.
+ *
+ * @returns {Error|undefined}
+ * @private
+ */
+const closeMainClient = async () => await _closeClientBase(mainClient);
```

Comment:
would remove base as suffix

Reply:
I always use base when it's usable with both main and subscriber client and I like the explicitness.